addons/languages/kurdish/pack/src/main/res/xml/kurmanji_qwerty.xml

```diff
@@ -1,58 +1,57 @@
-        <Key android:codes="101" android:keyLabel="e" ask:mShiftedCodes="68" ask:shiftedKeyLabel="D"/>
-        <Key android:codes="114" android:keyLabel="r" ask:mShiftedCodes="81" ask:shiftedKeyLabel="Q"/>
-        <Key android:codes="116" android:keyLabel="t" ask:mShiftedCodes="83" ask:shiftedKeyLabel="S"/>
-        <Key android:codes="121" android:keyLabel="y" ask:mShiftedCodes="88" ask:shiftedKeyLabel="X"/>
-        <Key android:codes="117" android:keyLabel="u" ask:mShiftedCodes="84" ask:shiftedKeyLabel="T"/>
-        <Key android:codes="105" android:keyLabel="i" ask:mShiftedCodes="72" ask:shiftedKeyLabel="H"/>
-        <Key android:codes="111" android:keyLabel="o" ask:mShiftedCodes="78" ask:shiftedKeyLabel="M"/>
-        <Key android:codes="112" android:keyLabel="p" ask:mShiftedCodes="80" ask:shiftedKeyLabel="P" android:keyEdgeFlags="right"/>
+<Keyboard android:keyWidth="9.089994%p"
+  xmlns:android="http://schemas.android.com/apk/res/android" xmlns:ask="http://schemas.android.com/apk/res-auto">
+    
+    <Row android:keyWidth="10.000002%p" android:keyHeight="@integer/key_short_height" android:keyEdgeFlags="left">
+        <Key android:codes="49" android:keyLabel="1" />
+        <Key android:codes="50" android:keyLabel="2" />
+        <Key android:codes="51" android:keyLabel="3" />
+        <Key android:codes="52" android:keyLabel="4" />
+        <Key android:codes="53" android:keyLabel="5" />
+        <Key android:codes="54" android:keyLabel="6" />
+        <Key android:codes="55" android:keyLabel="7" />
+        <Key android:codes="56" android:keyLabel="8" />
+        <Key android:codes="57" android:keyLabel="9" />
+        <Key android:codes="48" android:keyEdgeFlags="right" android:keyLabel="0" />
     </Row>
-    -->
-    <!-- original -->
     <Row>
-        <Key android:codes="113" android:popupCharacters="1" android:keyEdgeFlags="left"/>
-        <Key android:codes="119" android:popupCharacters="2"/>
-        <Key android:codes="101" android:popupCharacters="êé3"/>
-        <Key android:codes="114" android:popupCharacters="ȓ4"/>
-        <Key android:codes="116" android:popupCharacters="5"/>
-        <Key android:codes="121" android:popupCharacters="6"/>
-        <Key android:codes="117" android:popupCharacters="û7"/>
-        <Key android:codes="105" android:popupCharacters="î8"/>
-        <Key android:codes="111" android:popupCharacters="9"/>
-        <Key android:codes="112" android:popupCharacters="0" android:keyEdgeFlags="right"/>
+        <Key android:codes="113" android:popupCharacters="1" android:keyEdgeFlags="left" />
+        <Key android:codes="119" />
+        <Key android:codes="101" />
+        <Key android:codes="114" />
+        <Key android:codes="116" />
+        <Key android:codes="121" />
+        <Key android:codes="117" />
+        <Key android:codes="105" />
+        <Key android:codes="111" />
+        <Key android:codes="112" android:keyEdgeFlags="right" />
+        <Key android:codes="251" />
```

Comment:
keyedgeflag should be on this key instead of the one above.

Reply:
that for the special words that need to be on the right of layout

<a href='https://postimages.org/' target='_blank'><img src='https://i.postimg.cc/L6QSytT8/Capture.png' border='0' alt='Capture'/></a>